src/main/scala/com/fulcrumgenomics/bam/api/SamRecord.scala

```diff
@@ -269,6 +270,10 @@ trait SamRecord {
       SamPairUtil.getPairOrientation(this) == PairOrientation.FR
   }
 
+  def isConsensus: Boolean = {
+    AllPerReadTags.exists(this.contains)
+  }
+
```

Comment:
I would prefer this not on the `SamRecord` API since it is specific to the consensus tooling and not SAM records in general. The way this is implemented, it will return true if the SAM record is an fgbio-style duplex consensus, but would return false if it was an fgbio-style single strand consensus. I'd suggest moving the helper function to another consensus-specific namespace like `Umis.scala` and the logic could be simpler with pseudo-code: `hasTag(cd) | (hasTag(ad) and hasTag(bd))` so it would detect either a single-strand consensus or a duplex consensus.

Reply:
I think change the name as well to something like `isConsensusRead` because we'll lose some context.

Should we be looking for `cd`, `ad`, and `bd` (per base tags) or `cD`, `aD`, and `bD` (per read tags)? Or does it not matter?